app/assets/javascripts/application.js

```diff
@@ -49,4 +49,7 @@ $(document).ready(function() {
     $('body').smoothScroll({
         delegateSelector: 'a.smoothscroll'
     });
+    $(function () {
```

Comment:
Please don't to this in the application.js 
``
For performance reasons, the Tooltip and Popover data-apis are opt-in, meaning you must initialize them yourself.
``